Makefile

```diff
@@ -98,6 +101,13 @@ manifests:
 goveralls:
 	${DO} "TRAVIS_JOB_ID=${TRAVIS_JOB_ID} TRAVIS_PULL_REQUEST=${TRAVIS_PULL_REQUEST} TRAVIS_BRANCH=${TRAVIS_BRANCH} ./hack/build/goveralls.sh"
 
+checkformat:
+	## Just run gofmt without attempting to update files
+	@gofmt -l -s $(SOURCE_DIRS) | grep ".*\.go"; if [ "$$?" = "0" ]; then exit 1; fi
+lint:
+	## Run golint on source files
+	@$(foreach dir,$(SOURCE_DIRS),echo "loop dir: $(dir)\n" && golint "$(dir)/...";)
```

Comment:
ditto the above comment.  Also, if you think that it'd be preferred to run all the checks at once, a single script that combines them all would be fine.

Reply:
I'm open to however folks would like to see this.  I was following the goveralls pattern (ie using the makefile directive) but moving the lint and format checks into the format script makes sense if everybody's good with that.

I would leave the actual format option in the Makefile to run "gofmt -s -w" though if that makes sense?  Maybe it needs a more distinctive name like "make runformatter" ?

Reply:
Yup, that makes sense to me.

In case you haven't run into it yet, there's a generated file that fails `go vet`.  An issue was raised in kube about it but nothing has come of it yet
```
pkg/client/clientset/versioned/fake/clientset_generated.go:56: literal copies lock value from fakePtr: kubevirt.io/containerized-data-importer/vendor/k8s.io/client-go/testing.Fake
```

Reply:
Ahh, I haven't but thanks for the heads up.  I did notice that there's something wonky with setting the CDI_DIR when running `make format`, it fails on my local machine because I'm not building in docker (dir is always set to /go/src/kubevirt.io/....).  Will see if I can get that sorted.

Reply:
That's odd, it shouldn't be acting like it.  The idea is that CDI_DIR always reflects repo root, regardless of the env / parent dir path.